doc/source/index.rst

```diff
@@ -327,6 +327,27 @@ Options:
   - ``min_valid_data_fraction: 10`` - only generate products if at least 10% of covered
   part of scene contains valid data.
 
+Uploading produced data to S3
```

Comment:
I could like to amend this section with a disclaimer that this is NOT the recommended way to dispatch files. The dispatcher should be used instead.

Reply:
A note added in 3c32b26 .